cli/src/commands/lock_files.rs

```diff
@@ -56,10 +56,10 @@ pub fn get_packages_from_lockfile(path: &str) -> Result<(Vec<PackageDescriptor>,
         "package-lock.json" => parse::<PackageLock>(path)?,
         "yarn.lock" => parse::<YarnLock>(path)?,
         "requirements.txt" => parse::<PyRequirements>(path)?,
-        "Pipfile.txt" | "Pipfile.lock" => parse::<PipFile>(path)?,
+        "Pipfile" | "Pipfile.lock" => parse::<PipFile>(path)?,
         "poetry.lock" => parse::<Poetry>(path)?,
         "effective-pom.xml" => parse::<Pom>(path)?,
-        "gradle-dependencies.txt" => parse::<GradleDeps>(path)?,
+        "gradle.lockfile" => parse::<GradleDeps>(path)?,
```

Comment:
We could in theory keep support for our old detection mechanism by just adding the new formats. However I feel like that would be difficult for people to follow when reading this in the future.

So unless there's any evidence that these formats actually exist in practice, I'd rather remove support for the old naming scheme entirely.

Reply:
Thanks a lot for pointing that out. The existing file indeed seems to be somewhat odd for usage as a lock file.

Are you familiar with the reasoning behind the current Java Gradle support? If so, what would you suggest moving forward?

Seems like our best choice might be to keep it as `gradle-dependencies.txt` and add an issue about supporting the `gradle.lockfile`? It might also be worth it to expand our existing documentation about supported lockfiles with what all these are specifically used for, since that would likely also help us understand what each of these is and which one might be incorrectly named.